libs/ui/src/diagnostics/configuration_section.test.tsx

```diff
@@ -123,3 +123,27 @@ test('election, mark threshold provided', () => {
   screen.getByText(`Mark Threshold: 0.07`);
   screen.getByText(`Write-in Threshold: 0.05`);
 });
+
+test('election, mark threshold properly truncated', () => {
+  const { electionDefinition } = electionPrimaryPrecinctSplitsFixtures;
+  render(
+    <ConfigurationSection
+      electionDefinition={electionDefinition}
+      electionPackageHash="test-election-package-hash"
+      expectPrecinctSelection
+      precinctSelection={{
+        kind: 'SinglePrecinct',
+        precinctId: 'precinct-c1-w1-1',
+      }}
+      markThresholds={{
+        definite: 0.070707,
+        marginal: 0.05,
+        writeInTextArea: 0.05050505,
+      }}
+    />
+  );
+
+  screen.getByText(`Precinct: Precinct 1`);
+  screen.getByText(`Mark Threshold: 0.0707`);
+  screen.getByText(`Write-in Threshold: 0.0505`);
```

Comment:
Actually, maybe this will match even in the case of the non-truncated value showing. Will consider alternative tomorrow

Reply:
`getByText` will look for an exact match unless you use a regexp or pass additional args: https://testing-library.com/docs/queries/about/#textmatch-examples

So this is fine. You could also make your test value nonrepeating eg 0.1234567

Reply:
With those examples I don't feel convinced this will work as behaved. Because whether the screen says `Mark Threshold: 0.0707` or `Mark Threshold: 0.07070707`, I can anticipate `Mark Threshold: 0.0707` matching them both, as that substring is an exact match for both. It seems regex could actually be better, to indicate the end of the string. Sorry if I'm overcomplicating it!

And I don't think the nonrepeating number would change this

Reply:
Talked offline; my read of the docs is `getByText` performs an exact string match so `Mark Threshold: 0.0707` will not match `Mark Threshold: 0.07070707`.

Nonrepeating numbers is just a small readability improvement and catches any bugs in `truncate` that would result in eg. truncating the first part of decimal instead of the last.

Reply:
Can confirm @kshen0's understanding was correct, as `getByText` performs full string match. Tested [here](https://app.circleci.com/pipelines/github/votingworks/vxsuite/17182/workflows/6879182f-247b-43f7-a4af-e57a6375af62/jobs/679277), which showed that `Mark Threshold: 0` was not picked up when the screen showed `Mark Threshold: 0.0707`. So, I will leave the test as is, except for changing the number pattern to be non-repeating.